.github/workflows/test-models.yml

```diff
@@ -157,7 +157,7 @@ jobs:
     - name: Validate Stable Diffusion Models (Windows)
       if: matrix.suite == 'vulkan' && matrix.os == '7950x'
       run: |
-        ./setup_venv.ps1
+        ./setup_venv.ps1 --force
```

Comment:
shouldn't need this. This should be a new venv ?

Reply:
Makes sense -- I think I was trying to get the right torch-mlir version but it was a different issue.
Removed this change.